Modules/Devices/test/Devices.API.Tests.Integration/Models/ResponseData.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Net;
+
+namespace Devices.API.Tests.Integration.Models
+{
+    public class ResponseData
+    {
+        public HttpStatusCode ResponseStatus { get; set; }
+    }
+}
```

Comment:
Use file scoped namespace (I just noticed that it is wrong in multiple files - check all of them (you can do so via a global search, no need to open each file)

```suggestion
namespace Devices.API.Tests.Integration.Models;

public class ResponseData
{
    public HttpStatusCode ResponseStatus { get; set; }
}
```

Reply:
Should I apply this change to all files in the backbone?

Reply:
There shouldn't be a lot of them. I fixed it in all files during the monolith migration. But you can fix all occurrences, yes